src/pages/wallet/add.custom.token.js

```diff
@@ -122,6 +123,15 @@ class AddCustomToken extends Component {
       } catch (error) {
         const { addNotification } = this.props;
         const notification = getErrorNotification(error.code, 'button.retry') || getDefaultErrorNotification();
+
+        if (!getErrorNotification(error.code)) {
```

Comment:
Same about reporting the error unconditionally, sorry for spamming 😄

Reply:
No worries about spamming, let's move this discussion to the code block above and if we make a change, I'll address it in all places.